dts/riscv/ite/it8xxx2.dtsi

```diff
@@ -1138,6 +1150,7 @@
 			interrupt-parent = <&intc>;
 			interrupts = <160 IRQ_TYPE_LEVEL_HIGH>;
 			status = "disabled";
+			clocks = <&ecpm IT8XXX2_ECPM_CGCTRL3R BIT(3) 1>;
```

Comment:
Shouldn't this be:
```
clocks = <&ecpm IT8XXX2_ECPM_CGCTRL3R BIT(3) BIT(6)>;
```
Bit 6 of the CGCTRL3R register is marked as "always write 1 to this bit".

Reply:
Definitely yes, sir. My rookie mistake that I didn't modify the props field accordingly. I'll fix it ASAP.

Reply:
Definitely, sir. It should be BIT(6) here:
`clocks = <&ecpm IT8XXX2_ECPM_CGCTRL3R BIT(3) BIT(6)>;`